### PR TITLE
Add get projects command and implement --output command flag

### PR DESCRIPTION
**Pull request by depwaswho94**
Fixes https://github.com/devopsctl/gitlabctl/issues/4
And https://github.com/devopsctl/gitlabctl/issues/56

### CODE REVIEW

**Comment by bzon** [comment]
Fixes https://github.com/devopsctl/gitlabctl/issues/4

File: cmd/get_projects_test.go [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+package cmd
+
+import (
+	"testing"
+)
+
+func TestLsProjects(t *testing.T) {
+	setBasicAuthEnvs()
+	tt := []struct {
+		name string
```
Comment (bzon) [inline cmd/get_projects_test.go]: Use the test `name` field

File: cmd/flag_helpers.go [hunk 2]
```diff
@@ -45,6 +45,34 @@ func addGetGroupsFlags(cmd *cobra.Command) {
...
+		"Limit by projects owned by the current user")
+	cmd.Flags().Bool("membership", false,
+		"Limit by projects that the current user is a member of")
+	cmd.Flags().Bool("starred", false,
+		"Limit by projects starred by the current user")
+	cmd.Flags().Bool("statistics", false,
+		"Include project statistics")
+	cmd.Flags().String("visibility", "private",
+		"Limit by visibility public, internal, or private")
+	cmd.Flags().Bool("with_issues_enabled", false,
```
Comment (bzon) [inline cmd/flag_helpers.go]: Use dashes `-` instead of underscores `_`. Let's be consistent with the flags 👍

File: cmd/get_projects.go [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+// THE SOFTWARE.
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	gitlab "github.com/xanzy/go-gitlab"
+)
+
+var getProjectsCmd = &cobra.Command{
```
Comment (bzon) [inline cmd/get_projects.go]: This command has a lot of flags with different output. Can you provide at least 3 examples using [cobra.Command.Examples](https://godoc.org/github.com/spf13/cobra#Command) string?

```go
Examples: `
# display projects only with issues enabled
gitlabctl get projects --with-issues-enabled

# simple view
gitlactl get projects --simple
`
```

File: cmd/get_projects_test.go (hunk 1 above)
Comment (depwaswho94) [reply cmd/get_projects_test.go]: Done

File: cmd/flag_helpers.go (hunk 2 above)
Comment (depwaswho94) [reply cmd/flag_helpers.go]: Done

File: cmd/get_projects.go (hunk 3 above)
Comment (depwaswho94) [reply cmd/get_projects.go]: Done

**Comment by bzon** [comment]
Clos

**Pull request closed without merging**